endpoint/endpoint.go

```diff
@@ -38,12 +39,78 @@ func (ttl TTL) IsConfigured() bool {
 	return ttl > 0
 }
 
+// Targets is a representation of a list of targets for an endpoint.
+type Targets []string
+
+// NewTargets is a convenience method to create a new Targets object from a vararg of strings
+func NewTargets(target ...string) Targets {
```

Comment:
Is the point here to create a copy list of strings into another slice ? otherwise you could also simply do 
```
func NewTargets(target ...string) Targets {
 	return target
}
```
Or even `Targets([]string{"1.1.1.1"})`

Reply:
Yeah, the goal was to actually make copy. I read that there is still discussion about how to store DNS names (with or without trailing dot) internally. This can give the opportunity to modify the target to our internally used format, without modifying the original slice.
But I don't really know how useful this is or can be.

Reply:
At this point in the code, I think it is safe not to create a copy and use whatever is generated by sources (trailing dots are trimmed at that stage). It is DNS provider's responsibility to add or remove trailing dots when they are applied.